.github/workflows/velox.yml

```diff
@@ -0,0 +1,76 @@
+name: Velox Nightly Build and Test
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  container:
+    runs-on: ubuntu-latest
+    container: prestocpp/velox-velox-torcharrow:kpai-20220301
+    steps:
+      - name: Print CPU info
+        run: cat /proc/cpuinfo
+
+      - name: Setup Python environment
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Check out source repository
+        uses: actions/checkout@v2
+        with:
+            submodules: true
+
+      - name: Update submodules
+        run: |
+          git submodule update --init --recursive
+          cd csrc/velox/velox/
+          git submodule update --remote --merge
```

Comment:
can we just use the Velox commit pinned in TorchArrow? -- we can update daily, for example.

Reply:
Sure, we can do that; I thought we wanted to build against latest Velox nightly this this change. If we dont want to do that then I will remove this.

Reply:
Yeah... but the published TorchArrow wheel is a bit weird -- it's not a TorchArrow nightly build. 

I guess we need Velox Python library (long term plan, similar to PyArrow) to build Velox nightly wheel.